src/org.xtuml.bp.ui.graphics/src/org/xtuml/bp/ui/graphics/figures/ShapeImageFigure.java

```diff
@@ -76,6 +76,9 @@ protected void fillShape(Graphics graphics) {
 			return;
 		}
 		Color defaultColor = specification.getInternal();
+		if(Activator.getDefault().isDarkTheme()) {
+			defaultColor = Activator.getDefault().getColor(81, 86, 88);

```

Comment:
The color is modeled and loaded into the GD_ES instances from PEI data (defined in plugin.xml). If we are going to do this, let's properly model it. Perhaps we can add a new attribute for the dark mode color and then access the correct attribute based on the current mode.